src/Response.php

```diff
@@ -204,6 +204,15 @@ public function writeHead($status = 200, array $headers = array())
             );
         }
 
+        // assign date header if no 'date' is given, use the current time where this code is running
+        if (!isset($lower['date'])) {
+            // IMF-fixdate  = day-name "," SP date1 SP time-of-day SP GMT
+            $headers = array_merge(
+                array('Date' => gmdate('D, d M Y H:i:s') . ' GMT'),
+                $headers
+            );
```

Comment:
There was no particular reason. Must be my daily condition. Changed it in the newest commit.